app/views/videos/index.html.erb

```diff
@@ -1,25 +1,27 @@
 <div class="video-filter">
   <%= form_tag videos_path, :method => 'get', :class => 'form-search', :id => 'video-search' do %>
-    <div class="col-lg-8 col-md-6 no-padding">
-      <%= autocomplete_field_tag :search, nil, autocomplete_tag_name_videos_path, {
-        :id => 'search',
-        :class => 'form-control video-search',
-        :placeholder => 'Search by category or name.',
-        :'data-delimiter' => ' ',
-        :tabindex => '-1',
+    <div class="col-lg-6 col-md-5 no-padding">
+      <%= text_field_tag :search, nil, {
+        :class => 'form-control', :placeholder => 'Search by title or category.', :autocomplete => 'off'
```

Comment:
That placeholder has been shitty for a while. We're planning to remove search by subject once we finish case-insensitive search. We'll update placeholder along with it.